moved environment varialbles to env file #150

Shaunaksb opened the pull request:
- Removed environment variables from settings files
- Moved these variables to a new template file
- The template file is a temporary solution, until a method to handle keys is agreed upon
- To set up:
   - For windows: `copy env_template.txt .env`
   - For Mac/Linux: `cp env_template.txt .env`

- Issue #150

Shaunaksb committed:
moved environment varialbles to env file

saket-shukla reviewed (changes requested):
1. We should not need separate settings files now that we are using environment file.
2. The standard format for environment files is `.env` in the structure
```sh
KEY=VALUE
KEY2=VALUE
```
3. `.env` file should not be added to the source code.
4. Any access tokens and passwords shown here, need to be verified if they are still active. If they are, they should be reset/invalidated.
5. The file to commit is called `/env/sample` and it only contains all the keys without values for users to fill their own

Shaunaksb committed:
Configured Environment automation

Shaunaksb committed:
Merge branch 'develop' of https://github.com/Shaunaksb/SpotAxis into 150-replace-env-file

Shaunaksb commented:
- Added automation for configuring the environment with the scope of adding more automation such as configuring docker.
- Removed extra settings files, consolidated to on `settings.py` file
- Updated the key format in the `.env` file to as described

Shaunaksb committed:
restored fields that were mistakenly omitted in the previous fields

Shaunaksb commented:
Closing this PR in favour of a new one with no merge conflicts
